tests/book_spec.rb

```diff
@@ -0,0 +1,11 @@
+require '../classes/book'
```

Comment:
- [ ] I got an error when I tried to run the tests. See the screenshot below.  I believe it's a typo issue. It will be nice if you could investigate that and provide a fix.

See screenshot:
![a1](https://user-images.githubusercontent.com/39523312/157543442-e0d7534e-c92d-4ee8-8e7f-a93e2eeb92ec.png)

Reply:
Done